docs/install.md

```diff
@@ -61,6 +63,14 @@ sure you have [configured `kubectl` to access your cluster](https://kubernetes.i
      underlying disk type. For more info, see the section below on "creating a storage class
      manually".
 
+1. Examine the sample `values.yaml` in the `deploy-sourcegraph` repo for additional starting
```

Comment:
How about moving this section to the end. I'd prefer that folks get something up and running first before customizing config. It also segues nicer into update.md that way.

Reply:
I think when I first wrote it, I wasn't aware of how to update the config, but also my initial pass didn't have an auth thing, so looking through values.yaml to find out I needed one was important to do early on. But yeah, it could move.